src/api/index.js

```diff
@@ -18,14 +18,16 @@ const config = new Conf();
  * Checks if the plugin/package exists on npm
  *
  * @param {String} plugin - The name of the plugin/package
- * @return {Promise} - Throws an error if the plugin/package doesn't exist
+ * @return {Promise} - True if the plugin is found
  */
 const checkOnNpm = plugin => new Promise(resolve => {
-  npmName(plugin).then(available => {
-    if (available) {
-      throw new Error(ERR_MODULE_NOT_FOUND);
+  npmName(plugin).then(notFound => {
```

Comment:
`notFound === true` means that exists

---

how `npm-name` module resolves whether a package exists or not is really confusing 😜